hbase-server/src/main/java/org/apache/hadoop/hbase/util/FSUtils.java

```diff
@@ -771,7 +770,7 @@ static public void addToHDFSBlocksDistribution(
   public static int getTotalTableFragmentation(final HMaster master)
   throws IOException {
     Map<String, Integer> map = getTableFragmentation(master);
-    return map != null && map.size() > 0 ? map.get("-TOTAL-") : -1;
```

Comment:
please update the javadocs for `getTableFragmentation` to say it can't return `null` (AFAICT it doesn't currently).

Reply:
That method says:

```
@return A map for each table and its percentage.
```

It does not say anything about returning a null value.  I don't think it needs to list everything it won't return.  I think it's appropriate as is.

Reply:
we're a large project with a lot of contributors. If an API does not say that it can't return null and the language spec allows a null to be returned, we have to account for it.

so either

1) change this code to account for a null

or 

2) update the javadoc so the method says it doesn't return null.

Reply:
I'll go with `update the javadoc so the method says it doesn't return null.`

Thanks.